src/css.js

```diff
@@ -255,7 +263,7 @@ jQuery.extend( {
 			if ( !hooks || !( "set" in hooks ) ||
 				( value = hooks.set( elem, value, extra ) ) !== undefined ) {
 
-				style[ name ] = value;
+				customProperty ? style.setProperty( name, value ) : style[ name ] = value;
```

Comment:
I'm quite surprised it passed our linter... cc @markelog 

Please change that to the regular `if-else`, the minifier will optimize it anyway.